tigerpath/majors_and_certificates/majors/GER_2018.json

```
@@ -0,0 +1,403 @@
+{
+  "type": "Major",
+  "name": "German",
+  "code": "GER",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-german",
+    "https://german.princeton.edu/programs/undergraduate/about-major"

```

Add the handbook as well
```suggestion
    "https://german.princeton.edu/programs/undergraduate/about-major",
    "https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf"
```